ckanext/aircan/lib/airflow.py

```
@@ -0,0 +1,153 @@
+        self._access_token = token
+        return token
+
+    def request(self, method: str, endpoint: str, **kwargs: Any) -> requests.Response:
+        if "timeout" not in kwargs:
+            kwargs["timeout"] = self.timeout
+
+        url = self._join_url(endpoint)
+
+        if self.server_type == "gcp":
+            if not self._authed_session:
+                raise RuntimeError("GCP session not initialized")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)